app/controllers/message_threads/bulk/exports_controller.rb

```diff
@@ -17,7 +17,12 @@ def edit
         authorize @export
 
         @message_threads = message_thread_policy_scope.where(id: @export.message_thread_ids).includes(:messages)
-        @message_types = ["default"] + @message_threads.flat_map do |thread|
+        @message_forms = @message_threads.map do |thread|
+          thread.metadata['fs_form_id']
```

Comment:
Toto nie je uplne pekne, ale pokym nemame v Core nieco ako form a type, tak to nebude krajsie.